.github/workflows/sonar-analysis-on-merge.yml

```
@@ -1,16 +1,16 @@
 name: Sonar Analysis on Merge
```

```suggestion
name: Pull Request CI - Sonar
```
and update the filename please